doc/api/errors.md

```diff
@@ -970,172 +962,166 @@ made to mark a stream and dependent of itself.
 <a id="ERR_HTTP2_UNSUPPORTED_PROTOCOL"></a>
 ### ERR_HTTP2_UNSUPPORTED_PROTOCOL
 
-Used when `http2.connect()` is passed a URL that uses any protocol other than
-`http:` or `https:`.
+`http2.connect()` was passed a URL that uses any protocol other than `http:` or
```

Comment:
`..was passed to..`

Reply:
I think it's `was passed` because the URL is passed as an argument to `http2.connect()`.